index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="/styles/reset.css" />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <div
+      class="three-column-grid__expand-two gap-sm center-wrapper main-layout"
+    >
+      <header class="main-header three-row-grid__expand-two gap-sm">
+        <div class="box-md"></div>
+        <nav>
+          <ul class="nav-list">
+            <li>
+              <a class="nav-list--link" href="#">
+                <div class="frame">
+                  <img src="/assets/home.png" alt="home icon" />
+                </div>
+                <div class="link-text">Home</div>
+              </a>
+            </li>
+            <li>
+              <a class="nav-list--link" href="#">
+                <div class="frame">
+                  <img src="/assets/email.png" alt="message icon" />
+                </div>
+                <div class="link-text">Messages</div>
+              </a>
+            </li>
+            <li>
+              <a class="nav-list--link" href="#">
+                <div class="frame">
+                  <img
+                    src="/assets/notification.png"
+                    alt="notification icon"
+                  />
+                </div>
+                <div class="link-text">Notifications</div>
+              </a>
+            </li>
+          </ul>
+        </nav>
+        <div class="box-md">
+          <button>
+            <div class="icon"></div>
+            <div class="button-text">Tweet</div>
+          </button>
+        </div>
+      </header>
+      <main>
+        <!-- DOES IT BREAK THE NATURAL FLOW? WILL THE CONTENT PROVIDE THE HEIGHT? -->
+      </main>
+      <aside class="resources-sidebar">
+        <!-- DOES IT BREAK THE NATURAL FLOW? WILL THE CONTENT PROVIDE THE HEIGHT? -->
+      </aside>
```

Comment:
I don't like this class name. @dogezen or @shahzadnaeem do you have a suggestion?

Reply:
The class represents the container for the whole aside, so something reflecting that would be good:

`.news-aside-container` or similar could be a suggestion

Reply:
I'd consider `.search-and-suggestions-aside` but also like @dogezen suggestion. Naming can be tricky and trying to be consistent helps.